pkgs/by-name/ap/apvlv/package.nix

```diff
@@ -0,0 +1,111 @@
+{
+  cmake,
+  copyDesktopItems,
+  ebook_tools,
+  fetchFromGitHub,
+  freetype,
+  ghostscript,
+  gtk3,
+  installShellFiles,
+  lib,
+  libepoxy,
+  libpthreadstubs,
+  libXdmcp,
+  libxkbcommon,
+  libxml2,
+  libxshmfence,
+  man,
+  nix-update-script,
+  pcre,
+  pkg-config,
+  poppler,
+  stdenv,
+  testers,
+  webkitgtk,
+  wrapGAppsHook,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "apvlv";
+  version = "0.5.0";
+
+  src = fetchFromGitHub {
+    owner = "naihe2010";
+    repo = "apvlv";
+    rev = "refs/tags/v0.5.0-final";
```

Comment:
This looks like a one-off version name so I think it's alright to hard-code, but should put a note on `version` stating `rev` will need to be manually updated to update the package in the future.

Reply:
Thanks for the review. I've just added a comment above version attribute.